sambaplayersdk/src/main/java/com/sambatech/player/SambaPlayer.java

```diff
@@ -48,6 +48,7 @@
 import com.sambatech.player.plugins.PluginManager;
 import com.sambatech.player.utils.Helpers;
 import com.sambatech.player.utils.Orientation;
+import com.sambatech.player.utils.SharedPrefsUtils;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.sambatech.player.utils.SharedPrefsUtils'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.sambatech.com/coding_rules#rule_key=squid%3AUselessImportCheck)